ui/admin/app/templates/onboarding/quick-setup/create-resources/success.hbs

```diff
@@ -35,34 +35,50 @@
           {{t 'onboarding.success.links.learn'}}
         </a>
 
+        <div class='controller'>
```

Comment:
This class names feels a bit generic.  Should we namespace onboarding class names?  e.g. `onboarding-quick-start-`?  Open to ideas here.

Reply:
Yeah that is a good point. So something like `onboarding-quick-start-controller` and `onboarding-quick-start-developer`? This would prevent styles accidentally being applied to other "onboarding" work if we ever have more.

Reply:
Class names updated.